translations/ui-users/en.json

```diff
@@ -265,6 +265,7 @@
   "information.selectPatronGroup": "Select patron group",
   "information.patronGroup.users": "Users",
   "information.patronGroup.expirationOffset": "Expiration date offset (days)",
+  "information.patronGroup.expirationOffset.error": "Must be a integer > 0",
```

Comment:
Let's include an "allow" empty phrase here too, e.g.  

> Must be empty or an integer > 0

Reply:
That makes sense! Changes that.